lib/money_column/active_record_hooks.rb

```diff
@@ -5,11 +5,13 @@ def self.included(base)
     end
 
     module ClassMethods
```

Comment:
If they don't have one it's kind of scary. By the looks of the docs they can use procs for `constructor` to use a default currency for those columns. But we can follow up on this later.